### DISCUSSION TITLE
HTML within EditableDialogBoxConfiguration // Select boxes

### DISCUSSION THREAD

**Discussion by manuakasam** [pimcore/pimcore#7961] [Q&A / Support] [answered]
Hey guys, 
we're in the process of turning all our hacked component configurations over to the new EditableDialogBoxConfiguration. However I've noticed that it appears to be impossible to have any form of HTML as either keys or values within select boxes of the config. Here's a simple example:

    [
        'type'   => 'select',
        'label'  => 'Icon',
        'name'   => 'foo-',
        'width'  => 200,
        'config' => [
            'store' => [
                ['<svg ....>', 'Cirle <svg ....>'],
                ['<svg ....>', 'Triangle <svg ....>'],
                ['<svg ....>', 'Square <svg ....>'],
            ],
            'defaultValue' => '<svg ....>',
            'reload'       => true,
        ],
    ],

Icons and such are actually a pretty common thing to have as dropdowns and I'M wondering how we'd be able to achieve this given the new default configuration options for pimcore.

Is it possible to achieve this somehow? Maybe there's an option I don't know of to disable HTML escape or enable RAW html values?

Thanks upfront :)

**Comment by manuakasam** [comment]
I'm going to push this once in hope for help. Even digging through the source code I can't exactly make out where the issue happens. It appears that this is an error of (possibly) faulty json_encoding but i'm honestly not sure. 

Furthermore I do believe that this is quite a common real world scenario for websites (assign icon to link / button) so there should be a way to have HTML values within select elements. Can't really make CEOs and their secretaries go through the manual each time they need to select an icon to check for the correct name...

**Comment by dvesh3** [comment, marked as answer]
@manuakasam this is fixed for next release 6.8.8. please see https://github.com/pimcore/pimcore/pull/8061

**Reply by manuakasam** [reply]
Thank you, I didn't find this issue as I was searching for the interface name mostly. Looks like I wasn't too far off with my idea but didn't quite understand the correct place for the fix. Much appreciated!
